src/Map.js

```diff
@@ -116,4 +121,23 @@ const ViewChanger = ({center}) => {
     return null;
 }
 
+const  Legend = ({metadata}) => {
+    const map = useMap()
+    useEffect(() => {
+      if (map && metadata.updated_at) {
+        const legend = new L.Control({ position: "bottomleft" });
+  
+        legend.onAdd = () => {
+          const div = L.DomUtil.create("div");
+          div.id = "mapbox-legend"
+          div.innerHTML = `<h4>Last Updated: ${new Date(metadata.updated_at * 1000).toLocaleString()}</h4>`;
```

Comment:
Having to manually manipulate innerHTML feels a little suspicious to me. I suspect there may be a better way to do this with the react-leaflet library, but moreso I bet there's a way we can accomplish this without needing to do that, either.

In general one should probably not need to touch innerHTML or similar things inside a react project.